src/js/libgeo/GeoOps.js

```diff
@@ -2353,6 +2359,9 @@ geoOps.TrAffine.updatePosition = function(el) {
 geoOps.TrSimilarity = {};
 geoOps.TrSimilarity.kind = "Tr";
 geoOps.TrSimilarity.signature = ["P", "P", "P", "P"];
+geoOps.TrSimilarity.initialize = function(el) {
+    el.isEuclidean = 1;
```

Comment:
Please correct me if i'm wrong but similarity transformations should be non-euclidean?